src/components/Camera/CameraView.js

```diff
@@ -36,10 +36,15 @@ const CameraView = ( { camera, device }: Props ): Node => {
   const isActive = isFocused && isForeground;
 
   // used for pinch and double tap to zoom
+  const [tap, setTap] = useState( true );
```

Comment:
Here I go again with the naming: can we make this `doubleTapping` since it's tracking the status of a double tap event and nothing else? That way there's no wondering if it's a verb-like function or it represents a single tap status.

Reply:
Ok, poor form on part, but I did this in the interest of merging the other PR that depends on this one and in getting a build otu.